source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_OpenGLDebugDraw.cpp

```diff
@@ -186,15 +186,15 @@ RAS_OpenGLDebugDraw::RAS_OpenGLDebugDraw()
 		attribVector(trans, stride, offsetof(RAS_DebugDraw::Box2d, m_trans), 4, 1);
 	}
 
-	glBindVertexArray(0);
+	GPU_bind_vertex_array(0);
```

Comment:
For a goodie function you can declare `GPU_unbind_vertex_array()` To bind the null VAO.